modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/CheckProductionDumps_conf.pm

```diff
@@ -0,0 +1,627 @@
+        'db_type'                => 'core',
+        'out_file_stem'          => undef,
+        'xrefs'                  => 0,
+
+        ## fasta parameters
+        # types to emit
+        'dna_sequence_type_list' => [ 'dna' ],
+        'pep_sequence_type_list' => [ 'cdna', 'ncrna' ],
+        # Do/Don't process these logic names
+        'process_logic_names'    => [],
+        'skip_logic_names'       => [],
+        # Previous release FASTA DNA files location
+        # Previous release number
+        'prev_rel_dir'           => '/nfs/ftp/ensemblftp/ensembl/PUBLIC/pub/',
+
+        ## assembly_chain parameters
+        #  default => ON (1)
+        'compress'               => 1,
+        'ucsc'                   => 1,
+
+        ## BLAT
+        # A list of vertebrates species for which we have Blast server running with their associated port number
+        # We use this hash to filter our species which we don't need BLAT data and to have the port number in the file name
+        'blat_species'           => {
+            'homo_sapiens'                     => 30001,
+            'mus_musculus'                     => 30081,
+            'danio_rerio'                      => 30003,
+            'gallus_gallus'                    => 30005,
+            'bos_taurus'                       => 30017,
+            'oryctolagus_cuniculus'            => 30025,
+            'oryzias_latipes'                  => 30026,
+            'sus_scrofa'                       => 30039,
+            'meleagris_gallopavo'              => 30082,
+            'anas_platyrhynchos_platyrhynchos' => 30066,
+            'ovis_aries'                       => 30068,
+            'oreochromis_niloticus'            => 30073,
+            'gadus_morhua'                     => 30080,
+            'canis_lupus_familiaris'           => 30083,
+            'rattus_norvegicus'                => 30084,
+        },
+        # History file for storing record of datacheck run.
+        history_file             => undef,
+        datacheck_output_dir     => undef,
+        ## Indexing parameters
+        'skip_blat'              => 0,
+        'skip_ncbiblast'         => 0,
+        'skip_blat_masking'      => 1,
+        'skip_ncbiblast_masking' => 0,
+        ## Indexing parameters
+        'skip_convert_fasta'     => 0,
+
+    };
+}
+
+sub pipeline_create_commands {
+  my ($self) = @_;
+
+  return [
+    @{$self->SUPER::pipeline_create_commands},
+    'mkdir -p ' . $self->o('ftp_dir'),
+  ];
+}
+
+# Ensures output parameters gets propagated implicitly
+sub hive_meta_table {
+  my ($self) = @_;
+
+  return {
+    %{$self->SUPER::hive_meta_table},
+    'hive_use_param_stack' => 1,
+  };
+}
+
+sub pipeline_wide_parameters {
+  my ($self) = @_;
+
+  return {
+    %{$self->SUPER::pipeline_wide_parameters},
+    'pipeline_name' => $self->o('pipeline_name'),
+    'base_path'     => $self->o('ftp_dir'),
+    'release'       => $self->o('release'),
+  };
+}
+
+sub resource_classes {
```

Comment:
overriding resource class would prevent us from using this pipeline, say with SLURM. 
Can't we just integrate those at a higher level in the hierarchy?